FiveCalls/FiveCalls/FetchIssuesOperation.swift

```diff
@@ -11,15 +11,17 @@ import Foundation
 class FetchIssuesOperation : BaseOperation {
     
     let location: UserLocation?
+    let shouldFetchAllIssues: Bool
     
     // Output properties.
     // Once the job has finished consumers can check one or more of these for values.
     var httpResponse: HTTPURLResponse?
     var error: Error?
     var issuesList: IssuesList?
 
-    init(location: UserLocation?) {
+    init(location: UserLocation?, shouldFetchAllIssues: Bool) {
```

Comment:
Not sure I'm a fan of a boolean for this. Something a little less prescriptive (from the caller's perspective) might look like this:

```swift
enum IssueQuery {
    case all
    case top
}
```

Then the constructor could accept this type instead, so we'd call it like this:

```swift
FetchIssuesOperation(userLocation, query: .top)

// or

FetchIssuesOperation(userLocation, query: .all)
```

Going even further, perhaps we could move the URL creating into the enum, so the `.all` case would append the `inactive=true` query param.

Not sure if the last part fits, but just spitballing.

Doing this would give us flexibility to add more query variations in the future as well without modifying the constructor parameters.